Assignment3/CarlosJM7-y160190/CarlosJM7-CarlosJimenez.txt

```diff
@@ -0,0 +1,177 @@
+Carlos Jiménez Martín - 160190
+
+
+1. Get all the properties that can be applied to instances of the Politician class
```

Comment:
Nice, you have also interpreted that we wanted the properties where politicians are in the range. It looks ok.